docs/index.md

```diff
@@ -97,6 +97,21 @@ Now in order to access protected api urls you must include the `Authorization: B
 $ curl -H "Authorization: Bearer <your_token>" http://localhost:8000/protected-url/
 ```
 
+In addition to adding the Authorization: Bearer in your requests, make sure that you have decorated your views properly. For this, import the JSONWebTokenAuthentication authenticaion class from rest_framework_jwt.authentication
```

Comment:
> JSONWebTokenAuthentication authenticaion

"authenticaion" is misspelled, and the JSONWebTokenAuthentication can be styled as `JSONWebTokenAuthentication`

> make sure that you have decorated your views properly

I suggest you point out that this only applies to function based views. The class based views include this by overriding the `authentication_classes` class attribute. Feel free to add an example for the class based views as well.

Reply:
Sure! I will modify the PR according to your suggestions!